src/main/java/nextstep/subway/section/Sections.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.subway.section;
+
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+@Embeddable
+public class Sections {
+
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL)
+    List<Section> sections = new ArrayList<>();
+
+    public Sections() {
+    }
+
+    public void add(Section section) {
+        sections.add(section);
+    }
+
+    public boolean contains(Section section) {
+        return sections.contains(section);
+    }
+
+    public List<Section> getAllSection() {
+        return sections;
+    }
+
+    public List<Station> sortedStations() {
+        LinkedList<Station> stations = new LinkedList<>();
```

Comment:
linkedList의 기능을 사용하는게 아닌데 구현체의 타입을 쓸 필요가 있나요?

```suggestion
        List<Station> stations = new LinkedList<>();
```

Reply:
아 원래 `LinkedList`의 `addFirst()`나 `addLast()`를 사용할 생각이었는데, 이번 단계에서는 결국 사용하지 않았네요! `ArrayList`로 변경하였습니다.